core/trace/dt_span_serializer.go

```diff
@@ -23,6 +23,7 @@ import (
 	"google.golang.org/protobuf/proto"
 
 	"github.com/dynatrace-oss/opentelemetry-exporter-go/core/internal/version"
+	"github.com/dynatrace-oss/opentelemetry-exporter-go/core/semconv"
```

Comment:
```suggestion
	"github.com/dynatrace-oss/opentelemetry-exporter-go/core/internal/semconv"
```

We should not expose these names to users. Users should just use the public OTel semantic conventions.

Reply:
Done